pkg/types/dsse/dsse.go

```diff
@@ -61,7 +62,7 @@ func ToLogEntry(ds *pb.DSSERequestV0_0_2) (*pb.DSSELogEntryV0_0_2, error) {
 			allPubKeyBytes[v] = cert.RawBytes
 		}
 	}
-	signerVerifiers, err := verifyEnvelope(allPubKeyBytes, ds.Envelope)
+	signerVerifiers, err := verifyEnvelope(allPubKeyBytes, ds.Envelope, crypto.SHA256, algorithmRegistry)
```

Comment:
nit, can you add a TODO(https://github.com/sigstore/rekor-tiles/issues/176) for this